controllers/articles-controllers.js

```diff
@@ -12,13 +13,12 @@ exports.getArticleById = (req, res, next) => {
 
 exports.getArticles = (req, res, next) => {
   let { topic } = req.query;
-  // if (topic) topic = topic.toLowerCase();
-
-  // if (topic && !/^[a-z-_]+$/.test(topic)) {
-  //   return next({ status: 400, msg: "Invalid Topic Query" });
-  // }
-  selectArticles(topic)
-    .then((articles) => {
+  if (topic) topic = topic.toLowerCase();
```

Comment:
I'm not sure if we need to keep this line of code in. If someone was to put in an uppercase valid topic, you're right in that it would cause a 404, but it will be picked up with the error handling you currently have set up, so i don't think there's any need to manipulate it beforehand.